### PR TITLE
Fix dark mode issues in GVA

### PR DESCRIPTION
**Pull request by rasmustautsglia**
Some of the views were rendered differently in dark mode, and needed some adjustments.

### CODE REVIEW

File: GliaWidgets/Sources/View/Chat/GVA/Gallery/GalleryList/GvaGalleryListView.swift [hunk 1]
```diff
@@ -66,6 +66,10 @@ final class GvaGalleryListView: BaseView {
         ).activate()
     }
 
+    override func defineLayout() {
+        collectionView.backgroundColor = UIColor.clear
```
Comment (yurii-glia) [inline GliaWidgets/Sources/View/Chat/GVA/Gallery/GalleryList/GvaGalleryListView.swift:70]:
```swift
super.defineLayout()
```
missed super call.

File: GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift [hunk 2]
```diff
@@ -59,6 +59,18 @@ final class QuickReplyView: BaseView {
             collectionView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor)
         ])
     }
+
+    override func layoutSubviews() {
+        switch style.backgroundColor {
```
Comment (gersonnoboa) [inline GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift:64]: Doesn't layoutSubviews also need super? @yurii-glia

File: GliaWidgets/Sources/View/Chat/GVA/Gallery/GalleryList/GvaGalleryListView.swift (hunk 1 above)
Comment (rasmustautsglia) [reply GliaWidgets/Sources/View/Chat/GVA/Gallery/GalleryList/GvaGalleryListView.swift:71]: fixed [b613e53](https://github.com/salemove/ios-sdk-widgets/pull/715/commits/b613e53732030c573604cc51bab8c4b2c449012e)

File: GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift (hunk 2 above)
Comment (rasmustautsglia) [reply GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift:64]: It does indeed

Comment (rasmustautsglia) [reply GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift:65]: fixed [5c4854f](https://github.com/salemove/ios-sdk-widgets/pull/715/commits/5c4854fa6272308185fc86bdb7928d659cf978fa)

**Review by EgorovEI — APPROVED** [review]

**Review by gersonnoboa — APPROVED** [review]

File: GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift (hunk 2 above)
Comment (yurii-glia) [reply GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift:65]: for consistency yes, however, afaik, Apple doc for `layoutSubviews` method mentions that it is not necessary to call super for this method.

**Review by igorkravchenko — APPROVED** [review]

**Comment by rasmustautsglia** [comment]
!squash

**Comment by rasmustautsglia** [comment]
!merge

**Pull request merged**